packager/media/formats/mp4/mp4_muxer.cc

```diff
@@ -429,6 +430,36 @@ bool MP4Muxer::GenerateVideoTrak(const VideoStreamInfo* video_info,
   VideoSampleEntry video;
   video.format =
       CodecToFourCC(video_info->codec(), video_info->h26x_stream_format());
+  if (video.format == FOURCC_av01) {
+    auto av1_color_info =
```

Comment:
having to parse these back out of the codec string doesn't feel like the right solution